### PR TITLE
Add plugins SpeechBubble

### PR DESCRIPTION
**Pull request by sybri**
Hi,

I've create a plugin.

### CODE REVIEW

**Comment by jieter** [comment]
note that [your demo](https://sybri.github.io/demo/Leaflet.SpeechBubble/demo.html) only shows the speech bubble for me after moving the map.


Apart from that, the page loads `http://gc.kis.v2.scr.kaspersky-labs.com/7DC4751E-7850-A94F-913E-BFA8CA091E06/main.js` which not allowed by chrome:

```
demo.html:1 Mixed Content: The page at 'https://sybri.github.io/demo/Leaflet.SpeechBubble/demo.html' was loaded over HTTPS, but requested an insecure script 'http://gc.kis.v2.scr.kaspersky-labs.com/7DC4751E-7850-A94F-913E-BFA8CA091E06/main.js'. This request has been blocked; the content must be served over HTTPS.
```

**Comment by sybri** [comment]
i've removed this script !! from a cut/paste

**Comment by sybri** [comment]
I've update the init phase and the demo as well.

File: docs/plugins.md [hunk 1]
```diff
@@ -1449,6 +1449,18 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/ggolikov">Grigory Golikov</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/sybri/Leaflet.SpeechBubble/">Leaflet.SpeechBubble</a>
+		</td>
+		<td>
+			Popup a speech bubble with the arrow that follow points,layer, markers ...
+			 (<a href="hhttps://sybri.github.io/demo/Leaflet.SpeechBubble/demo.html">demo</a>).
```
Comment (perliedman) [inline docs/plugins.md]: There's an extra `h` in `https` here!

**Comment by sybri** [comment]
corrected

**Comment by perliedman** [comment]
:+1:

**Pull request merged**